gulpfile.js

```diff
@@ -366,6 +374,9 @@ function createTestSource(testsName) {
         return null;
     }
     args.push('--browserManifestFile=' + PDF_BROWSERS);
+    if (process.env.GENERATE_COVERAGE) {
+      args.push('--browserManifestFile=' + PDF_BROWSERS);
```

Comment:
`--browserManifestFile` is a boolean flag. You shouldn't pass any value to this command-line flag.